assets/js/checkout.js

```diff
@@ -0,0 +1,800 @@
+/* global wc_checkout_params */
+jQuery( function( $ ) {
+    'use strict';
+
+    /**
+     * Object to handle Swedbank Pay checkout payment forms.
+     */
+    window.wc_sb_checkout = {
+        js_url: null,
+        paymentMenu: null,
+        isPaymentMenuLoaded: false,
+        xhr: false,
+
+        /**
+         * Initialize e handlers and UI state.
+         */
+        init: function( form ) {
+            this.form         = form;
+            this.form_submit  = false;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'form_submit' is not in camel case.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-woocommerce-payments/pullRequest?prid=5340032)